rpm/clang.spec

```
@@ -153,7 +153,7 @@ pushd build
 %endif
 	-DCLANG_INCLUDE_TESTS:BOOL=OFF \
 	-DLLVM_MAIN_SRC_DIR=%{_datadir}/llvm/src \
-%if 0%{?__isa_bits} == 64
+%ifarch aarch64
```

```suggestion
%if "%{_lib}" == "lib64"
    -DLLVM_LIBDIR_SUFFIX=64 \
%endif
```
This should be much better ^